analyzer/src/main/kotlin/curation/Sw360PackageCurationProvider.kt

```diff
@@ -62,8 +62,6 @@ class Sw360PackageCurationProvider(sw360Configuration: Sw360StorageConfiguration
                     PackageCuration(
                         id = pkgId,
                         data = PackageCurationData(
-                            declaredLicenses = sw360Release.embedded?.licenses
-                                ?.mapNotNullTo(sortedSetOf()) { it?.shortName },
```

Comment:
We cannot drop this without a substitute, but assign it to `concludedLicense` instead.